pkg/api/v1alpha1/file.go

```diff
@@ -0,0 +1,42 @@
+package v1alpha1
+
+const (
+	ApiVersion string = "jsonnet-filer.zeet.co/v1alpha1"
+	Kind       string = "File"
+)
+
+type ObjectMeta struct {
```

Comment:
would it be worth while to implement these using upstream types like 
```
import (
    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
)


type File struct {
	metav1.TypeMeta   `json:",inline"`
	metav1.ObjectMeta `json:"metadata,omitempty"`

	Content          any        `json:"content"`
	EncodingStrategy string     `json:"encodingStrategy"`
}
```

Reply:
Possible in a future iteration. Right now, I don't want to add fields that have no meaning, such as annotations, etc, nor do I want this to be strictly tied to k8s. (similarities help people though).